src/server/cmd/pachd/main.go

```diff
@@ -290,9 +289,6 @@ func doFullMode(config interface{}) (retErr error) {
 	}
 	env := serviceenv.InitWithKube(serviceenv.NewConfiguration(config))
 	debug.SetGCPercent(50)
-	go func() {
-		log.Println(http.ListenAndServe(fmt.Sprintf(":%d", env.PProfPort), nil))
-	}()
```

Comment:
Is this being removed for good or relocated. I think it probably makes sense to just remove it since there's now another way to get profiles out of it.

Reply:
It got relocated to the set of goroutines setup at the end of this function. What other way is there to get profiles?

Reply:
We expose profiles through the debug API, `pachctl debug pprof`. It saves users from having to muck around with ports, and works in settings like hub.

Reply:
K, will remove.

Reply:
Removing in a separate PR.